engine/src/main/scala/cromwell/server/CromwellRootActor.scala

```diff
@@ -224,6 +225,9 @@ abstract class CromwellRootActor(terminator: CromwellTerminator,
     "JobExecutionTokenDispenser"
   )
 
+  lazy val groupMetricsActor: ActorRef =
+    context.actorOf(GroupMetricsActor.props(EngineServicesStore.engineDatabaseInterface))
+
```

Comment:
This is where the singleton GroupMetricsActor is created and wired through other actors to make its way through to backend actor.